esmvaltool/cmorizers/obs/cmorize_obs_esacci_oc.py

```diff
@@ -45,17 +46,43 @@ def _fix_data(cube, var):
 
 def _add_depth_coord(cube):
     """Add depth auxiliary coordinate for CMIP5 standard."""
+    def adjust_dim(dim):
+        if dim == 0:
+            return dim
+        return dim + 1
```

Comment:
it's not good practice to have a function inside a function especially when this can be unpacked to an if conditional eg `if dim > 0: dim = dim + 1`